document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -1064,27 +1067,27 @@ protected Document prepareDocument() throws SQLException, IOException {
       return new JSONDocument(MAPPER.writeValueAsString(jsonNode));
     }
 
-    @SneakyThrows
     @Override
     public void close() {
-      resultSet.close();
+      // nothing to do here
```

Comment:
Let's have them `connection`, `preparedStmt` and `ResultSet`, so we can close it here.